src/tasks/trelloTasks.ts

```diff
@@ -0,0 +1,145 @@
+import type { SapphireClient } from "@sapphire/framework";
+import { ColorResolvable, MessageEmbed } from "discord.js";
+import dotenv from "dotenv";
+
+const fetch = require('node-fetch');
+const membersJson = require("../../json/members.json")
+const listsToIgnoreJson = require("../../json/listsToIgnore.json")
+
+dotenv.config()
+
+// super useful types that we'll be using
+export type Member = {trelloId: string, discordId:string, name:string};
```

Comment:
can we make a extra file somewhere for types? I find that easier to look through instead. something like src/typings/index.ts, and you put all of your types in there

Reply:
kk ill do that in a diff pr tho